picard/ui/newuserdialog.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright (C) 2023 Bob Swift
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+
+from PyQt5 import QtWidgets
+
+from picard.const import PICARD_URLS
+
+
+class NewUserDialog():
+
+    def __init__(self):
+
+        dialog_text = _(
+            (
+                "<h2 align=center>READ THIS BEFORE USING PICARD</h2>"
+                "<p>"
+                "Picard is a very flexible music tagging tool, using MusicBrainz metadata for tags and file naming, "
+                "but if you don't understand how it works, what its limitations are, and how to configure it, "
+                "you can easily mess up your music library."
+                "</p><p>"
+                "We therefore <strong>STRONGLY</strong> recommend that you:"
+                "</p><ol>"
+                "<li>Read the Picard documentation at <a href='{documentation_url}'>{documentation_url}</a> "
```

Comment:
```suggestion
                "<li>Read the Picard  <a href='{documentation_url}'>documentation</a> "
```

could work, though I noticed in dark theme here, link color is hard to read (that's another issue) and we can't set a tooltip (title attribute doesn't work).

![image](https://github.com/metabrainz/picard/assets/151042/7d430cc9-2feb-4adc-86d7-1f58f44e68c0)

So not sure about this.

Reply:
I actually prefer it as you've suggested.  As you say, the issue with links on the dark theme is another matter, and shouldn't impact how we decide what text to display.